src/main/java/frc/robot/Robot.java

```diff
@@ -20,6 +20,9 @@
 import frc.robot.constants.AutoConstants;
 import frc.robot.constants.ArmConstants.ArmState;
 
+import edu.wpi.first.cameraserver.CameraServer;
+//This import allows the usage of cameras connected to the RoboRIO.
```

Comment:
Can you move this above the import? By convention, you generally put comments above the thing you're explaining rather than below.